cdap-ui/server/config/themes/default.json

```diff
@@ -0,0 +1,29 @@
+{
+  "spec-version": "1.0",
+  "styles": {
+    "brand-primary-color": "#3b78e7",
+    "navbar-color": "#333333",
+    "font-family": "-apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, 'Helvetica Neue', Arial, sans-serif"
+  },
+  "content": {
+    "logo": {
+      "type": "link",
+      "arguments": {
+         "url": "/cdap_assets/img/company_logo.png"
+      }
+    },
+    "footer-text": "Licensed under the Apache License, Version 2.0",
+    "footer-link": "https://www.apache.org/licenses/LICENSE-2.0"
+  },
+  "features": {
+    "dashboard": "true",
+    "reports": "true",
+    "data-prep": "true",
+    "pipelines": "true",
+    "analytics": "true",
+    "rules-engine": "true",
+    "metadata": "true",
+    "hub": "true",
+    "footer": "true"
```

Comment:
this should be just a simple boolean, not text. JSON definition accept a boolean value